code/modules/client/preference_setup/loadout/loadout_augments.dm

```diff
@@ -0,0 +1,110 @@
+/datum/gear/augment
+	display_name = "integrated PDA"
+	description = "An augment that allows the user to access a remote PDA system."
+	path = /obj/item/organ/internal/augment/pda
+	augment = TRUE
+	cost = 3
+	sort_category = "Augments"
+	whitelisted = list("Human", "Off-Worlder Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Aut'akh Unathi", "Skrell", "Baseline Frame", "Hephaestus G1 Industrial Frame", "Hephaestus G2 Industrial Frame", "Xion Industrial Frame", "Zeng-Hu Mobility Frame", "Bishop Accessory Frame")
```

Comment:
this should be available to standard unathi as well. not all unathi are sk'akh or si'akh.

Reply:
I already pointed this out on the lore writer channels and the unathi lore writer did not oppose it.